packages/shared/components/popups/SendFormPopup.svelte

```diff
@@ -48,6 +53,22 @@
         isTagInputOpen = true
     }
 
+    let tagError: string = ''
+    function validateTag(): Promise<void> {
+        if (tag.length > 64) {
+            tagError = localize('error.send.tagTooLong')
+            return Promise.reject(tagError)
+        }
+    }
+
+    let metadataError: string = ''
+    function validateMetadata(): Promise<void> {
+        if (metadata.length > 8192) {
+            metadataError = localize('error.send.metadataTooLong')
+            return Promise.reject(metadataError)
```

Comment:
Does it make sense to throw the errors you used for deep links here instead of rejecting the promise?

Reply:
these errors are used for alerts and logging, but here it should only display the error under the input. i think it would require a larger change